galleon-plugins/src/main/java/org/wildfly/galleon/plugin/WfInstallPlugin.java

```diff
@@ -845,4 +850,54 @@ private static void mkdirs(final WildFlyPackageTasks tasks, Path installDir) thr
             }
         }
     }
+
+    private static void changeLineEndings(final WildFlyPackageTasks tasks, final Path installDir) throws ProvisioningException {
+        // If not line end filters are present no need to walk the directory
+        if (tasks.getUnixLineEndFilters().isEmpty() && tasks.getWindowsLineEndFilters().isEmpty()) {
+            return;
+        }
+        try {
+            Files.walkFileTree(installDir, new SimpleFileVisitor<Path>() {
+                @Override
+                public FileVisitResult visitFile(final Path file, final BasicFileAttributes attrs) throws IOException {
+                    final String relative = installDir.relativize(file).toString();
+                    for (FileFilter filter : tasks.getUnixLineEndFilters()) {
+                        if (filter.matches(relative)) {
+                            changeLineEndings(file, false);
+                        }
+                    }
+                    for (FileFilter filter : tasks.getWindowsLineEndFilters()) {
+                        if (filter.matches(relative)) {
+                            changeLineEndings(file, true);
+                        }
+                    }
+                    return FileVisitResult.CONTINUE;
+                }
+            });
+        } catch (IOException e) {
+            throw new ProvisioningException(e);
```

Comment:
Would you mind adding a little message as well? Saying that it has failed to adjust the line endings as requested?

Reply:
No problem. I only didn't because it seemed other messages were coming from the `org.jboss.galleon.Errors` and I wasn't sure if that's where all messages are supposed to live.